src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,459 @@
+  content="1,114",
+  classes="big-number",
+  sheets_gid="1681733418",
+  sql_file="consoleLog_unused_preload.sql"
+)
+}}
+
+When using an incorrect `as` attribute value - as opposed to unrecognised value, such as using `style` instead of `script`; the browser will duplicate the file download as the request won't match the resource stored in the preload cache.
+
+<p class="note">Note: While <code>video</code> is included in the spec, it isn't supported by any browser and would be treated as an invalid value and ignored.</p>
+
+#### Unused font files
+
+More than 5% of pages which preload font files, preload more font files than needed. When preloading font files, all browsers which support `preload` also support `.woff2`. This means that, assuming that the `.woff2` font files are available, it is not necessary to preload older formats, including `.woff`.
+
+### Third Parties
+
+You are able to use resource hints to connect to or download resources from both first and [third parties](./third-parties). While `dns-prefetch` and `preconnect` are only useful when connecting to different origins, including subdomains; `preload` and `prefetch` may be used for both resources on the same origin and resources hosted by third parties.
+
+When considering which resource hints you should use for third party resources, you need to evaluate the priority and role of each third party on your application's loading experience and whether the costs are justified. 
+
+Prioritising third party resources over your own content is potentially a warning sign, however there are cases when this is recommended. As an example, if we look at cookie notice scripts; which are required in the European Union by General Data Protection Regulation; these are usually accompanied by a `dns-prefetch` or `preconnect` script as they are highly obtrusive to the user experience and also a prerequisite for some site functions, such as serving personalised ads.
+
+<figure markdown>
+| host                       | dns-prefetch | preconnect | preload | Total       | 
+|----------------------------|--------------|------------|---------|-------------| 
+| adservice.google.com       | 0.25%        | 0.37%      | 42.72%  | 43.34%      | 
+| fonts.gstatic.com          | 0.82%        | 25.45%     | 0.74%   | 27.01%      | 
+| fonts.googleapis.com       | 14.09%       | 4.59%      | 3.05%   | 21.72%      | 
+| s.w.org                    | 20.00%       | 0.25%      | -       | 20.25%      | 
+| cdn.shopify.com            | -            | 1.82%      | 12.16%  | 13.98%      | 
+| www.google-analytics.com   | 1.27%        | 4.05%      | 0.17%   | 5.49%       | 
+| www.googletagmanager.com   | 1.94%        | 3.14%      | 0.25%   | 5.32%       | 
+| siteassets.parastorage.com | -            | -          | 5.24%   | 5.24%       | 
+| images.squarespace-cdn.com | -            | 4.51%      | -       | 4.51%       | 
+| ajax.googleapis.com        | 2.30%        | 1.79%      | 0.36%   | 4.45%       | 
+
```

Comment:
Can you convert to HTML like above? I find it best to grab HTML from Dev Tools and then add `class="numeric"` attributes.